README.md

```diff
@@ -154,6 +154,23 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## DevMode
+In the devmode a user can edit and reload the files using the quarkus devmode
+The external files that can be edited during the dev mode
+must be saved in a configmap per type, currently CamelRoute, OpenAPI and Async API are supported,
+to reference the configmaps in the labels section must be added using the following keys and your preferred name as values,
+the same label must be added on the configmap. 
+Currently the SWF app handle Camel, OpenAPI and AsyncAPI, to handle other type is up to the user add the dependencies to handle other types
+
+```
+labels:
+.. 
+  ext_camel: mycamel-configmap
+  ext_openapi: myopenapi-configmap
+  ext_asyncapi: myasyncapi-configMap
+```
+Every entry in these configmaps will be turned into a file inside /src/main/resources and using the key as file name
+
```

Comment:
Nice! But let's add the documentation to devmode to the guide instead. We can save this text to use later.

I'll start a draft PR.